platform/ios/CHANGELOG.md

```diff
@@ -12,6 +12,16 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 * Added a one-time warning about possible attribute loss when initializing an `MGLShapeSource` with an `MGLShapeCollection` [#12625](https://github.com/mapbox/mapbox-gl-native/pull/12625)
 * Added an `-[MGLMapViewDelegate mapView:shapeAnnotationIsEnabled:]` method to specify whether an annotation is selectable. ([#12352](https://github.com/mapbox/mapbox-gl-native/pull/12352))
 
+## 4.0.5 - August 15, 2018
```

Comment:
Being that 4.0.5 and 4.3.0 are on the same date, please sort 4.0.5 below 4.3.0. 🙇